packages/docs/docs/docs/upgrading.md

```diff
@@ -160,6 +160,24 @@ truffle(local)> myContract.x().then(x => x.toString())
 43
 ```
 
+Now, just as a mental exercise, let's imagine that instead of just adding a new 
+function to the contract (a change to functionality), we wanted to add a new 
+variable `t` (a change to storage layout). How do we set the initial value of `t`?
+The variables `x` and `s` were initialized with the `initialize` function,
+which was called when the proxy was created via the `zos create MyContract --init initialize --args ...` 
+command. The `update` command also accepts `--init` and `--args` parameters, so we can use a function
```

Comment:
I'd first explain that, even if we should add `t` to the initialize function (so we can initialize it on new instances we create), we cannot call that function again on contracts that have already been created. Then go on to explain that we need a new function, and only then mention that `zos update` also accepts the init option.